addons/hr_attendance/report/attendance_errors.py

```diff
@@ -51,15 +37,28 @@ def _lst_total(self, employee_id, dt_from, dt_to, max, *args):
             if abs(r['delay'].seconds) < max*60:
                 total2 += r['delay']
 
-        result_dict = {
-                'total': total and str(total).split('.')[0],
-                'total2': total2  and str(total2).split('.')[0]
-                }
+        result_dict = {'total': total and str(total).split('.')[0],
+                       'total2': total2 and str(total2).split('.')[0]}
         return [result_dict]
 
-
-class report_hr_attendanceerrors(osv.AbstractModel):
-    _name = 'report.hr_attendance.report_attendanceerrors'
-    _inherit = 'report.abstract_report'
-    _template = 'hr_attendance.report_attendanceerrors'
-    _wrapped_report_class = attendance_print
+    @api.multi
+    def render_html(self, data):
+        Report = self.env['report']
+        report = Report._get_report_from_name('hr_attendance.report_attendanceerrors')
+        employees = self.env['hr.employee'].browse(data.get('emp_ids'))
```

Comment:
Its generic report, so you'll have document ids, you'll not have data here, printing report throws traceback, you have to prepare data based on current recordset(i.e. self)